src/content/docs/iast/exploitable-vulns.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: New Relic IAST Exploitable Vulnerabilities
+tags:
+    - IAST
+    - New Relic IAST
+    - Exploitable vulnerabilities
+metaDescription: Learn how to detect exploitable vulnerabilities with New Relic IAST.
+---
+
+import iastExplVulns from 'images/iast_screenshot-crop_exploitable-vulns.webp'
+
+import iastVulnsDetails from 'images/iast_screenshot-crop_vuln-details.webp'
+
+
+The New Relic IAST exploitable vulnerabilities page informs you about the total number of critical and high vulnerabilites you have in your tested applications.
+
+To open the page, go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Exploitable Vulnerabilities** on the left navigation pane.
+
+    <img
+        title="Overview of the exploitable vulnerabilities found in your application."
+        alt="Overview of the exploitable vulnerabilities found in your application."
+        src={iastExplVulns}
+    />
+
+    <figcaption>
+        Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Exploitable Vulnerabilities** on the left navigation pane.
+    </figcaption>
+
+## Details and remediation of a vulnerability [#remediation-vuln]
+
+New Relic IAST provides accurate out-of-the-box results with detailed vulnerability descriptions and detections down to the line of code.
+
+To see the details and the remediation instructions of a vulnerability:
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** and click **Exploitable Vulnerabilities** on the left navigation pane.
+
+2. In the **Detected exploitable vulnerabilities** table, select an exploitable vulnerability to explore its details and understand how to address it. You can search the detected vulnerabilities by type, file name, or application name. You can also order each column alphabetically.
+
+    <img
+        title="Details of a vulnerability."
+        alt="Details of a vulnerability."
+        src={iastVulnsDetails}
+    />
+
+3. The **Understanding the exploitable vulnerability** section gives you information about the details, the stack trace, and the HTTP request of the vulnerability.
+
+4. Expand the **Run the exploit** section to copy the curl you'll find here.
+
+5. Once you've applied the suggested fix, rebuild your application, and rerun the New Relic IAST.
```

Comment:
shouldn't it be "rerun IAST"? I don't think we need to write "the New Relic IAST" every time